### ISSUE TITLE
React Native

### ISSUE THREAD

**Issue by Imaginativeone** [jameskerr/react-arborist#288]
Is it possible to use Arborist in a React Native app? If not, please direct me to a component that can be used in RN?

**Comment by Taimoor-Shahzad** [comment]
Yes, it is technically possible to use Arborist in a React Native app, but since Arborist is primarily designed for React (web) environments, it might not work seamlessly in React Native due to platform-specific differences. React Native focuses on mobile environments, and components built for the web often rely on DOM-specific features that are unavailable in React Native.

For a tree view or similar functionality in a React Native app, I recommend looking into native-compatible components like, React Native Tree Select, React Native Tree View.
These components are specifically designed for mobile and will integrate smoothly with your React Native project.

**Comment by blue86321** [comment]
I don't think so. I dig into the root component and found it's browser tag, which is not compatible to RN.

**Comment by Imaginativeone** [comment]
> I don't think so. I dig into the root component and found it's browser tag, which is not compatible to RN.

@blue86321 How do I find that tag? Perhaps I could work on an update.
